content/blog/using-jina-embeddings-haystack/index.md

```diff
@@ -0,0 +1,198 @@
+---
+layout: blog-post
+title: 'Using Jina Embeddings v2 with Haystack 2.0 pipelines to summarize legal documents'
+description: Learn how to use the Jina v2 Embedding models in a RAG pipeline with our new Haystack integration.
+featured_image: thumbnail.png
+images: ["blog/using-jina-embeddings-haystack/thumbnail.png"]
+alt_image: The logos for Haystack and Jina AI sit against a blue background with cartoonish art of a man and a woman holding a magnifying glass up to a large clipboard, next to a gavel.
+date: 2024-01-03
+last_updated: 2024-01-03
+authors:
```

Comment:
You can add `toc: True` attribute to generate a Table of Contents section right beneath the author section in article page. As a user, I find it quite useful to follow the content
```suggestion
toc: True
authors:
```